Makefile

```diff
@@ -42,3 +46,13 @@ rpm-in-mock-el7: srpm
 
 check-local:
 	ansible-playbook $(ANSIBLE_EXTRA_ARGS) -e config=$(CONFIG) -e subject=$(TEST_IMAGE_NAME) -e results=$(RESULTS) -e artifacts_dir=$(ARTIFACTS_DIR) ./local.yml -e setup=true
+
+clean:
+	git clean -dfx
+
+install: clean
```

Comment:
install requires clean and clean removes all untracked files, that's pretty bold